libff/CMakeLists.txt

```diff
@@ -79,11 +79,11 @@ if ("${IS_LIBFF_PARENT}")
     add_dependencies(check ${TEST_NAME})
   endfunction()
 
-  libff_test(test_common common/tests/test_common.cpp)
+  libff_test(test_common common/tests/test_common.cpp) # VV
```

Comment:
Could you remove all of these `VV` comments, especially where nothing has changed.

We don't have a policy about adding names to comments, but personally I'd be in favour of not putting names. i.e. I think we can add comments / todos as required and discuss them in reviews if they are controversial.